### ISSUE TITLE
Exception in thread "Skript offline player resolver thread (fetches UUIDs from the minecraft servers)" 

### ISSUE THREAD

**Issue by jewome62** [SkriptLang/Skript#3672]
### Description
I have exception when i reboot my server. And stop reboot process. I must kill the process for continue the reboot.

### Steps to Reproduce
I don't know why ...

### Expected Behavior
No block the reboot

### Errors / Screenshots
```
[22:21:16 WARN]: Exception in thread "Skript offline player resolver thread (fetches UUIDs from the minecraft servers)" java.lang.NoClassDefFoundError: Could not initialize class ch.njol.skript.bukkitutil.UnresolvedOfflinePlayer
[22:21:16 WARN]:        at ch.njol.skript.bukkitutil.UnresolvedOfflinePlayer$1.run(UnresolvedOfflinePlayer.java:60)
[22:21:16 WARN]:        at java.base/java.lang.Thread.run(Thread.java:834)
```

### Server Information
* **Server version/platform:** Tuinity 1.16.4
* **Skript version:** 2.3.2

**Comment by Pikachu920** [comment]
are you rebooting or reloading?

**Comment by Dancull47** [comment]
I just got the same error upon attempting to stop my server. 
`[00:38:03 INFO]: [LuckPerms] Disabling LuckPerms v5.1.93
[00:38:03 INFO]: [LuckPerms] Starting shutdown process...
[00:38:03 INFO]: [LuckPerms] Closing messaging service...
[00:38:13 WARN]: 	at ch.njol.skript.bukkitutil.UnresolvedOfflinePlayer$1.run(UnresolvedOfflinePlayer.java:60)
[00:38:13 WARN]: 	at java.base/java.lang.Thread.run(Thread.java:834)
[00:38:13 WARN]: Exception in thread "Skript offline player resolver thread (fetches UUIDs from the minecraft servers)" java.lang.NoClassDefFoundError: Could not initialize class ch.njol.skript.bukkitutil.UnresolvedOfflinePlayer`

**Comment by jewome62** [comment]
> are you rebooting or reloading?

Reboot of the server

**Comment by ShaneBeee** [comment]
> Skript version: 2.3.2

We are on Skript 2.5.3... we only offer support for the latest version. Can you please try replicate this issue on 2.5.3 and report back.

**Comment by jewome62** [comment]
> > Skript version: 2.3.2
> 
> We are on Skript 2.5.3... we only offer support for the latest version. Can you please try replicate this issue on 2.5.3 and report back.

Sorry this is a rewrite problem between hands and keyboard :P my version is 2.5.3
```
[11:31:07 INFO]: Skript version 2.5.3
[11:31:07 INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[11:31:07 INFO]: Website: https://skriptlang.github.io/Skript
[11:31:07 INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, FranKusmiruk, ShaneBeee, Blueyescat, JRoy, Nicofisi, APickledWalrus, xXAndrew28Xx, Syst3ms, TheLimeGlass, Pikachu920, Sashie, Wealthyturtle, OfficialDonut and eyesniper2
```

**Comment by Tanimsiz** [comment]
Same problem at 1.16.5 paper latest build

**Comment by SlimeDog** [comment]
The initially-reported issue occurs on server shutdown on Skript 2.5.3 with Paper 1.16.5-457

**Comment by jewome62** [comment]
Any news ?
My server didnot restart well everyday ...

**Comment by TheDGOfficial** [comment]
That class probably should be rewritten. The `java.lang.NoClassDefFoundError: Could not initialize class ...` errors means the ClassLoader (PluginClassLoader in this case) had an uncaught exception loading the class. It does not mean class is not found. May be confusing at first look, though. The real exception can be printed by just adding a try catch to the static initializer. However that class probably should be rewritten and/or removed completely from stratch as it caused issues in the past too, better of rewriting it completely with proper Java coding standards and usage of modern Java/Bukkit features.

For users, just wait until it is fixed or ignore the error if it happens during shutdown. If it happens on runtime/start-up though, offlineplayers on command arguments or such may cause issues.

**Comment by jewome62** [comment]
@TheDGOfficial 
> For users, just wait until it is fixed or ignore the error if it happens during shutdown. If it happens on runtime/start-up though, offlineplayers on command arguments or such may cause issues.

OK but the problem is the bug block the stop.
We must kill the server and the force save of the map is after unload of skript.
So i kill a server without save of map
And i can have corruption of maps

**Comment by SlimeDog** [comment]
Confirmed fixed: Skript 2.6-alpha1
